packages/Algernon.package/ALGCore.class/class/setQuickAccessModifier..st

```diff
@@ -0,0 +1,6 @@
+preferences
+setQuickAccessModifier: aString
+    ({'cmd'. 'ctrl'} includes: aString asLowercase)
+        ifTrue: [self quickAccessModifier: aString.
+            ^ true].
+    ^ false
```

Comment:
```suggestion
    ^ ({'cmd'. 'ctrl'} includes: aString asLowercase)
        ifTrue: [self quickAccessModifier: aString. true].
        ifFalse: [false]
```
It feels like this is easier to understand. But 🤷‍♂️

Reply:
Or could we maybe even remove the `ifFalse:` branch completely and it will still return false in this case because that's the default behaviour?

Reply:
What about

```suggestion
setQuickAccessModifier: aString
    | valid |
    valid := ({'cmd'. 'ctrl'} includes: aString asLowercase).
    valid ifTrue: [ self quickAccessModifier: aString].
    ^ valid
```